aws/aws.go

```diff
@@ -1,8 +1,9 @@
 package aws
 
 type AWS struct {
-	KMSKeyArn string `json:"kmsKeyArn" yaml:"kmsKeyArn"`
-	Masters   []Node `json:"masters" yaml:"masters"`
-	Region    string `json:"region" yaml:"region"`
-	Workers   []Node `json:"workers" yaml:"workers"`
+	KMSKeyArn  string   `json:"kmsKeyArn" yaml:"kmsKeyArn"`
+	Masters    []Node   `json:"masters" yaml:"masters"`
+	Region     string   `json:"region" yaml:"region"`
+	Workers    []Node   `json:"workers" yaml:"workers"`
+	PublicKeys []string `json:"publicKeys" yaml:"publicKeys"`
```

Comment:
I see. Shouldn't this be solved by the cert-operator somehow? I miss the picture there. Can you come up with a proposal of the certificate workflow or should we sort this out in a different way? IMO this is not different from the rest of the certificates we want to deal with. It is about access around the guest cluster. So we should handle certificates the same way, even if they are used for different kinds of access. As of now the aws-operator takes the test certificates as command line arguments. This should be later consumed as separate TPOs as well IIRC. So we should handle the certificates for EC2 instance access the same way. WDYT?